Assets/00_Content/Scripts/Interactables/Beers/BeerTap.cs

```diff
@@ -58,8 +80,9 @@ public class BeerTap : Interactable {
 					GameObject beer = Instantiate(beerPrefab);
 					itemSlot.PlaceItem(beer.GetComponent<PickUp>());
 
-					if (Tavern.Instance != null)
-						Tavern.Instance.SpendsMoney(beerData.cost);
+					BeerAmount -= 1;
+
+					if (Tavern.Instance != null) Tavern.Instance.SpendsMoney(beerData.cost);
```

Comment:
This can probably lead to a small problem, what would happen here if another player spends the rest of the taverns money while we are pouring? I think we should follow the same logic as with repairing tables => spend the money instantly when we start interacting and refund it if the player cancels the interaction.

Reply:
I hadn't thought of that, I'll look at how it is done in the table repair and see if I should implement something like that hre.